packages/govuk-frontend/src/govuk/components/exit-this-page/exit-this-page.mjs

```diff
@@ -391,6 +391,13 @@ export class ExitThisPage extends GOVUKFrontendComponent {
     }
   }
 
+  /**
+   * Name for the component used when initialising using data-module attributes.
+   *
+   * @type {string}
```

Comment:
Only other suggestion, but optional, would be removing these types

Where default values are obvious to the compiler already, don't need them

```suggestion
```